test/mocks/cdo/manage/cdo.js

```diff
@@ -0,0 +1,57 @@
+const noTasksStartedYet = {
+  tasks: {
+    applicationPackSent: {
+      key: 'applicationPackSent',
+      available: true,
+      completed: false,
+      editable: true,
```

Comment:
this will change to:

```
{
    key: 'applicationPackSent',
    available: true,
    completed: false,
    readonly: false,
    timestamp: undefined
}
```